task "npm run grunt:package -- test" fails

Here's the result after running the command line:
```
$ PACKAGE=plugin-mercury npm run grunt:package -- test:node

> spark-js-sdk@0.5.105 pregrunt:package /Users/tratu/projects/repo/ciscospark
> npm run check-package


> spark-js-sdk@0.5.105 check-package /Users/tratu/projects/repo/ciscospark
> if [ -z ${PACKAGE+x} ]; then echo 'environment variable PACKAGE must be defined' && exit 1; fi


> spark-js-sdk@0.5.105 grunt:package /Users/tratu/projects/repo/ciscospark
> grunt --gruntfile Gruntfile.package.js "test:node"

Running "env:default" (env) task

Running "env:secrets" (env) task

Running "mochaTest:integration" (mochaTest) task
Warning: init callback must be a function Use --force to continue.

Aborted due to warnings.


Execution Time (2016-08-08 13:08:18 UTC-7)
loading tasks          555ms  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 88%
env:default              7ms  ▇ 1%
mochaTest:integration   68ms  ▇▇▇▇▇▇▇▇ 11%
Total 633ms


npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/Users/tratu/.npm-packages/bin/npm" "run" "grunt:package" "--" "test:node"
npm ERR! node v5.8.0
npm ERR! npm  v3.9.6
npm ERR! code ELIFECYCLE
npm ERR! spark-js-sdk@0.5.105 grunt:package: `grunt --gruntfile Gruntfile.package.js "test:node"`
npm ERR! Exit status 3
npm ERR!
npm ERR! Failed at the spark-js-sdk@0.5.105 grunt:package script 'grunt --gruntfile Gruntfile.package.js "test:node"'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the spark-js-sdk package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt --gruntfile Gruntfile.package.js "test:node"
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs spark-js-sdk
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls spark-js-sdk
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/tratu/projects/repo/ciscospark/npm-debug.log
```


npm-debug.log
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node',
1 verbose cli   '/Users/tratu/.npm-packages/bin/npm',
1 verbose cli   'run',
1 verbose cli   'grunt:package',
1 verbose cli   '--',
1 verbose cli   'test:node' ]
2 info using npm@3.9.6
3 info using node@v5.8.0
4 verbose run-script [ 'pregrunt:package', 'grunt:package', 'postgrunt:package' ]
5 info lifecycle spark-js-sdk@0.5.105~pregrunt:package: spark-js-sdk@0.5.105
6 verbose lifecycle spark-js-sdk@0.5.105~pregrunt:package: unsafe-perm in lifecycle true
7 verbose lifecycle spark-js-sdk@0.5.105~pregrunt:package: PATH: /Users/tratu/.npm-packages/lib/node_modules/npm/bin/node-gyp-bin:/Users/tratu/projects/repo/ciscospark/node_modules/.bin:/usr/local/bin:/Users/tratu/.npm-packages/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/bin/jamf:/Users/tratu/.npm-packages/bin:/Users/tratu/.npm-packages/bin
8 verbose lifecycle spark-js-sdk@0.5.105~pregrunt:package: CWD: /Users/tratu/projects/repo/ciscospark
9 silly lifecycle spark-js-sdk@0.5.105~pregrunt:package: Args: [ '-c', 'npm run check-package' ]
10 silly lifecycle spark-js-sdk@0.5.105~pregrunt:package: Returned: code: 0  signal: null
11 info lifecycle spark-js-sdk@0.5.105~grunt:package: spark-js-sdk@0.5.105
12 verbose lifecycle spark-js-sdk@0.5.105~grunt:package: unsafe-perm in lifecycle true
13 verbose lifecycle spark-js-sdk@0.5.105~grunt:package: PATH: /Users/tratu/.npm-packages/lib/node_modules/npm/bin/node-gyp-bin:/Users/tratu/projects/repo/ciscospark/node_modules/.bin:/usr/local/bin:/Users/tratu/.npm-packages/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/bin/jamf:/Users/tratu/.npm-packages/bin:/Users/tratu/.npm-packages/bin
14 verbose lifecycle spark-js-sdk@0.5.105~grunt:package: CWD: /Users/tratu/projects/repo/ciscospark
15 silly lifecycle spark-js-sdk@0.5.105~grunt:package: Args: [ '-c', 'grunt --gruntfile Gruntfile.package.js "test:node"' ]
16 silly lifecycle spark-js-sdk@0.5.105~grunt:package: Returned: code: 3  signal: null
17 info lifecycle spark-js-sdk@0.5.105~grunt:package: Failed to exec grunt:package script
18 verbose stack Error: spark-js-sdk@0.5.105 grunt:package: `grunt --gruntfile Gruntfile.package.js "test:node"`
18 verbose stack Exit status 3
18 verbose stack     at EventEmitter.<anonymous> (/Users/tratu/.npm-packages/lib/node_modules/npm/lib/utils/lifecycle.js:245:16)
18 verbose stack     at emitTwo (events.js:100:13)
18 verbose stack     at EventEmitter.emit (events.js:185:7)
18 verbose stack     at ChildProcess.<anonymous> (/Users/tratu/.npm-packages/lib/node_modules/npm/lib/utils/spawn.js:24:14)
18 verbose stack     at emitTwo (events.js:100:13)
18 verbose stack     at ChildProcess.emit (events.js:185:7)
18 verbose stack     at maybeClose (internal/child_process.js:850:16)
18 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
19 verbose pkgid spark-js-sdk@0.5.105
20 verbose cwd /Users/tratu/projects/repo/ciscospark
21 error Darwin 15.6.0
22 error argv "/usr/local/bin/node" "/Users/tratu/.npm-packages/bin/npm" "run" "grunt:package" "--" "test:node"
23 error node v5.8.0
24 error npm  v3.9.6
25 error code ELIFECYCLE
26 error spark-js-sdk@0.5.105 grunt:package: `grunt --gruntfile Gruntfile.package.js "test:node"`
26 error Exit status 3
27 error Failed at the spark-js-sdk@0.5.105 grunt:package script 'grunt --gruntfile Gruntfile.package.js "test:node"'.
27 error Make sure you have the latest version of node.js and npm installed.
27 error If you do, this is most likely a problem with the spark-js-sdk package,
27 error not with npm itself.
27 error Tell the author that this fails on your system:
27 error     grunt --gruntfile Gruntfile.package.js "test:node"
27 error You can get information on how to open an issue for this project with:
27 error     npm bugs spark-js-sdk
27 error Or if that isn't available, you can get their info via:
27 error     npm owner ls spark-js-sdk
27 error There is likely additional logging output above.
28 verbose exit [ 1, true ]

```

--- a/Gruntfile.package.js
+++ b/Gruntfile.package.js
@@ -388,7 +388,7 @@ module.exports = function(grunt) {
       ]);
     }
 
-    if (parseInt(process.versions.node.split('.')[0]) < 5) {
+    if (parseInt(process.versions.node.split('.')[0]) <= 5) {
       return requires.concat([
         'clarify',
         function() {

